scl/slack/slack.conf

```diff
@@ -0,0 +1,65 @@
+#############################################################################
+# Copyright (c) 2018 Balabit
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
+
+@requires http
+@requires add-contextual-data
+
+# Throttle: https://api.slack.com/docs/rate-limits
+block destination slack(
+  hook-url()
+  author-template('"host: ${HOST} | program: ${PROGRAM}(${PID}) | severity: ${PRIORITY}"')
+  title('')
+  template('${MSG}')
+  colors-file('`scl-root`/slack/colors.csv')
+  throttle(1)
+  use-system-cert-store(yes)
+  ...)
+{
+  channel {
+    parser {
```

Comment:
Optional: I would like to propose using `list-nth` instead of `add-contextual-data`. I see the following benefits.
- One less module dependency.
- One less parser.
- Without parser, we do not need channel.
- If you expose color palette through parameter, users have easy and clean way to make their own coloring. (I know the same could be done through editing a separate file, but it is nicer to have every syslog configuration in one place.